packages/round-manager/src/features/api/__tests__/utils.test.ts

```diff
@@ -418,10 +413,38 @@ describe("generateApplicationSchema", () => {
     expect(schema).toEqual(
       expect.arrayContaining([
         expect.objectContaining({
-          question: "Email",
+          question: "Email Address",
           encrypted: true,
         }),
       ])
     );
   });
 });
+
+function createQuestions(
+  metadata: {
+    customQuestions: {
+      email?: string;
+      fundingSource?: string;
+      profit2022?: string;
+      teamSize?: string;
+    };
+  },
+  overrides?: { required: boolean; encrypted: boolean }
+): any {
+  const questions: any = [];
+  Object.keys(metadata.customQuestions).forEach((key) => {
+    questions.push({
+      id: Object.keys(metadata.customQuestions).indexOf(key),
+      question: humanReadableLabels[key],
+      type: inputTypes[key],
+      required: true,
+      info: metadata.customQuestions.email,
```

Comment:
why is `info` always set to `metadata.customQuestions.email` here? i know this is just a helper in a test file, but this wouldn't generate the correct expected data for the other list elements, right?

Reply:
resolved in [396398e](https://github.com/gitcoinco/grants-round/pull/452/commits/396398e258f31db230f952fdb3a48c352124f8ee)